<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Abi" failures="1" tests="9" name="com.testcase.PostEmployeerecord" time="0.055" errors="0" timestamp="28 May 2020 21:34:24 GMT" skipped="0">
  <testcase classname="com.testcase.PostEmployeerecord" name="getallheaders" time="0.005"/>
  <testcase classname="com.testcase.PostEmployeerecord" name="checkContenttype" time="0.008"/>
  <testcase classname="com.testcase.PostEmployeerecord" name="checkcookies" time="0.005"/>
  <testcase classname="com.testcase.PostEmployeerecord" name="checkStatusline" time="0.005"/>
  <testcase classname="com.testcase.PostEmployeerecord" name="checkContentEncoding" time="0.009"/>
  <testcase classname="com.testcase.PostEmployeerecord" name="checkResponsetime" time="0.005"/>
  <testcase classname="com.testcase.PostEmployeerecord" name="checkResponseBody" time="0.007"/>
  <testcase classname="com.testcase.PostEmployeerecord" name="checkServertype" time="0.004"/>
  <testcase classname="com.testcase.PostEmployeerecord" name="checkStatuscode" time="0.007">
    <failure message="expected [201] but found [200]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [201] but found [200]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:388)
	at org.testng.Assert.assertEquals(Assert.java:398)
	at com.testcase.PostEmployeerecord.checkStatuscode(PostEmployeerecord.java:54)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStatuscode -->
</testsuite> <!-- com.testcase.PostEmployeerecord -->
